tests/boulder-integration.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -xe
 # Simple integration test. Make sure to activate virtualenv beforehand
 # (source venv/bin/activate) and that you are running Boulder test
-# instance (see ./boulder-fetch.sh).
+# instance (see ./boulder-start.sh).
```

Comment:
I think `boulder-fetch.sh` is correct here. `boulder-start.sh` was removed in #4211.